lib/fluent/plugin/out_tdlog.rb

```diff
@@ -108,7 +110,6 @@ def format(tag, time, record)
         record.delete(:time) if record.has_key?(:time)
 
         if record.size > @key_num_limit
-          # TODO include summary of the record
```

Comment:
👍  getting rid of those TODOs.... 
I really don't get the meaning of TODOs in code once they are merged in :p